### PR TITLE
[Hotfix][E2E][Clickhouse] Fix the bug of clickhouse e2e case

### PR DESCRIPTION
**Pull request by TyrantLucifer**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by hailin0** [comment]
title：[Bug][E2E][Clickhouse] Fix clickhouse e2e testcase

**Comment by TyrantLucifer** [comment]
> title：[Bug][E2E][Clickhouse] Fix clickhouse e2e testcase

Done.

**Review by hailin0 — DISMISSED** [review]

**Review by dijiekstra — DISMISSED** [review]
LGTM

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseSinkCDCChangelogIT.java [hunk 1]
```diff
@@ -180,12 +186,15 @@ private void checkSinkTableRows() throws SQLException {
                 Arrays.asList(1L, "A_1", 100),
                 Arrays.asList(3L, "C", 100))
             .collect(Collectors.toSet());
-        Assertions.assertIterableEquals(expected, actual);
+        if (!Arrays.equals(expected.toArray(), expected.toArray())) {
```
Comment (wuchunfu) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseSinkCDCChangelogIT.java]: There seems to be a problem with the parameters here. It should be `if (!Arrays.equals(actual.toArray(), expected.toArray())) {`

**Review by CalvinKirs — APPROVED** [review]

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseSinkCDCChangelogIT.java (hunk 1 above)
Comment (TyrantLucifer) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseSinkCDCChangelogIT.java]: Done

**Review by wuchunfu — APPROVED** [review]
LGTM

**Pull request merged**